### ISSUE TITLE
npm support

### ISSUE THREAD

**Issue by shdpl** [uptrace/uptrace-js#132]
Uptrace-js version: v1.11.0
Pre-*b9c2b3e* works fine
```
root@7ba10f5ae2ce:/tmp/proj# npm --version
9.6.3
```

Problem:
```
root@7ba10f5ae2ce:/tmp/proj# npm install --save @uptrace/node
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

npm ERR! A complete log of this run can be found in: /root/.npm/_logs/2023-04-12T22_49_48_542Z-debug-0.log
```

*uptrace-node* _package.json_ depends on `"@uptrace/core": "workspace:*"` which npm doesn't seem to understand.

**Comment by vmihailenco** [comment]
@shdpl please try again. This should be fixed in v1.11.1

Thanks for reporting!

**Comment by shdpl** [comment]
Works fine, thanks for the quick feedback.
